lib/challenge.js

```diff
@@ -51,7 +67,12 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+some_pet = {
+  name : "Miley Cyrus",
+  species : "Capybara",
+  breed : "Scottish Biter",
+  noise : "Harumph"
+}
```

Comment:
I guess I made a mistake in taking it out, bc we have the constructor function after it (odd order) which I did include it in.